docs/testing/troubleshoot-development-errors.md

```diff
@@ -98,6 +98,13 @@ If this occurs, you can update the XSD files that Visual Studio uses to the late
 
 You can repeat the previous process for any additional schemas that are out-of-date.
 
+## When working offline, no Office APIs work
+
+When you're loading the Office JavaScript Library from a local copy instead of from the CDN, the APIs may stop working if the library isn't up-to-date. If you have been away from a project for a while, reinstall the library to get the latest version. The process varies according to your IDE:
```

Comment:
Per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26918) the sentence prior to a list should end in a period. How about this: 

```suggestion
When you're loading the Office JavaScript Library from a local copy instead of from the CDN, the APIs may stop working if the library isn't up-to-date. If you have been away from a project for a while, reinstall the library to get the latest version. The process varies according to your IDE. Choose one of the following options based on your environment.
```